README.md

```diff
@@ -166,3 +166,100 @@ To fix it, install [Kyma CLI manually](https://github.com/kyma-project/cli#insta
    export KYMA=$(which kyma)
    ```
 
+## Sourcecode linting
```

Comment:
The idea was to split developer documentation (in the repo README) from customer documentation (in the docs folder), right? Then I'd state the intended audience in each of the READMEs.

Meaning, add here at the beginning of the Overview who you're addressing and which goals this doc, the README, achieves.

Reply:
I moved developer docu under docs/contributor. The root readme should not classify an audience I suggest